tools/walletextension/wallet_extension.go

```diff
@@ -346,3 +352,46 @@ func adjustStateRoot(rpcResp interface{}, respMap map[string]interface{}) {
 		}
 	}
 }
+
+// handleSpecialRPCMethods checks method name in the request and if it is one of the methods that we use differently than
+// eth standard. At the moment it is only getStorageAt method. It returns response if we should use that response or nil otherwise
+func (w *WalletExtension) handleSpecialRPCMethods(request *accountmanager.RPCRequest, hexUserID string) map[string]interface{} {
```

Comment:
This is a method that intercepts the eth_getStorage ? Probably intercepterSomething would be a better name ? wdyt ?

Reply:
renamed